datalad_container/containers_run.py

```diff
@@ -37,9 +37,11 @@ class ContainersRun(Interface):
     # the rest is put in the verbose help and manpage
     """Drop-in replacement of 'run' to perform containerized command execution
 
-    Container(s) need to be configured beforehand (see containers-add).
-    If only one container is known, it will be selected automatically,
-    otherwise a specific container has to be specified.
+    Container(s) need to be configured beforehand (see containers-add). If no
+    container is specified and only one container is configured in the
+    top-level dataset, it will be selected automatically. If more than one
```

Comment:
"top-level" (which I read as a top most superdataset in current hierarchy) or "current"?

Reply:
> "top-level" (which I read as a top most superdataset in current hierarchy)

OK, I'll changed it to "current".